src/main/java/java/core/basesyntax/lib/Inject.java

```diff
@@ -0,0 +1,4 @@
+package java.core.basesyntax.lib;
+
+public @interface Inject {
```

Comment:
does this project work correctly?
should we add `@Retention(RetentionPolicy.RUNTIME)` here?